implementation/visualisation/src/main/java/org/inaetics/dronessimulator/visualisation/Settings.java

```diff
@@ -1,8 +1,5 @@
 package org.inaetics.dronessimulator.visualisation;
 
-/**
- * Created by langstra on 10-3-17.
- */
 public class Settings {
```

Comment:
I am not in favor of too long and descriptive class names, but I think that in this case Settings is not descriptive enough (settings for what)? It is kind of fine but I like a bit more description a bit better.